### ISSUE TITLE
Cannot create container for service webapp:  invalid bind mount spec

### ISSUE THREAD

**Issue by AnthonyCoughlin** [OpenDroneMap/WebODM#84]
All,

See attached screenshot.  Receiving an error after running the ./webodm.sh start' command.  Cannot create container for service webapp: invalid bind mount spec'.  There are additional warnings a few lines up as well.  Thanks for all the help thus far!

Anthony

![webodm_containererror2](https://cloud.githubusercontent.com/assets/25381671/22393897/f327035c-e4d6-11e6-959a-49e3f278dcd8.JPG)

**Comment by pierotofy** [comment]
@AnthonyCoughlin, could you try to run:

```
MSYS_NO_PATHCONV=1 ./webodm.sh start
```

?

**Comment by pierotofy** [comment]
Alternatively, also try:

```
COMPOSE_CONVERT_WINDOWS_PATHS=1 ./webodm.sh start
```

Do any of these two solve the problem?

**Comment by AnthonyCoughlin** [comment]
pierotofy,

See attached screenshot.  Thanks again, I was reading about the windows/msys path errors and conversion after searching the error.  

It seems as though it worked but now its freezing when trying to access /webodm/wait-for-it.sh.  I've been researching this for a bit but could use some assistance. 

 I'm not going to try to pretend here....  I'm super new to this!  So please understand how appreciative I am for all the help.

![webodm_wait-for-it_error](https://cloud.githubusercontent.com/assets/25381671/22400062/c99a2894-e570-11e6-8046-c675fee2afa0.JPG)

**Comment by AnthonyCoughlin** [comment]
Searching the error (bad interpreter:) - I've been seeing a dos2unix dependency that can be installed on docker.  Do you recommend this approach?

**Comment by pierotofy** [comment]
No worries :) I think we are all learning how to troubleshoot these errors at the moment.

Just to rule something out, did you modify wait-for-it.sh while troubleshooting previous problems? If so, try to revert it back by issuing `git checkout wait-for-it.sh`. If you have modified it on Windows, you probably have inadvertently changed the line endings, which are going to confuse bash.

**Comment by pierotofy** [comment]
Another cause could be that your git settings converted the line endings. Try to clone the repo with:

`git clone http://github.com/OpenDroneMap/WebODM --config core.autocrlf=input`

I think `dos2unix` could work, but it's one more dependency that I'd rather not install as part of the process. Let's explore some other options first.

**Comment by AnthonyCoughlin** [comment]
Ok, no dos2unix, got it.

I didn't modify anything that I know of.  My workflow after the initial error: connect to the WebODM folder and run the ./webodm.sh with the prefix you provided (COMPOSE_CONVERT_WINDOWS_PATHS=1 ./webodm.sh start).  That's when I received the 'bad interpreter' error.

***(For the assistance of others your first recommendation line of code didn't work for me - MSYS_NO_PATHCONV=1 ./webodm.sh start)***

I'll test the clone recommendation and let you know what happens!  Thanks again!

**Comment by AnthonyCoughlin** [comment]
pierotofy,

See attached.  Success!  Thanks again for your assistance.  Couldn't have done it with out you!

Workflow:
I deleted the existing \webodm directory using `rm -rf \webodm`
Re-cloned with `git clone http://github.com/OpenDroneMap/WebODM --config core.autocrlf=input`
Navigated to the webodm folder `cd \webodm`
Ran `COMPOSE_CONVERT_WINDOWS_PATHS=1 ./webodm.sh start`
WebODM started

CANNOT WAIT TO START PLAYING WITH THIS APP!!

Feel free to close this one out.

Anthony
